FrontEnd/src/UserAuthentication/Auth.js

```diff
@@ -42,8 +57,25 @@ const recoverPassword = async (email) => {
     const reset = sendPasswordResetEmail(auth, email);
```

Comment:
Why do we send a password reset email when a user is logging out?

Reply:
They are two separate functions